### ISSUE TITLE
Better default constructor for structs

### ISSUE THREAD

**Issue by vitreo12** [vitreo12/omni#21]
It would be great to have this:

```
struct Phasor:
    p : float

init:
   phasor = Phasor(0.0)
```

instead of the `new` construct:

```
struct Phasor:
    p : float

init:
   phasor = Phasor.new(0.0)
```

**Comment by vitreo12** [comment]
Writing `defs` or referring to `structs` by name should still be made possible:

```
def newPhasor() -> Phasor:
   return Phasor(0.0)
```

**Comment by vitreo12** [comment]
Possible fix in #70
